pyrene/src/components/DateTimePicker/RangeSelector/RangeSelector.tsx

```diff
@@ -0,0 +1,196 @@
+import React, {
+  useState,
+  ReactNode,
+} from 'react';
+import clsx from 'clsx';
+import Button from '../../Button/Button';
+import {
+  getFutureDate, standardEUDateFormat, standardEUTimeFormat,
+  convertToUTCtime, getDateTypeFromddmmyyyyWithSep, convertDateTypeToString, convertTimeTypeToString, getTimeTypeFromhhmmWithSep, convertToZoneTime,
+} from '../../../utils/DateUtils';
+import TimeRangeSelector from '../../TimeRangeSelector/TimeRangeSelector';
+import { CalendarContainer } from '../ReactDatePickerWrapper/ReactDatePickerWrapper';
+
+import styles from './RangeSelector.css';
+import DateTimePicker from '../DateTimePicker';
+import DetectClickOutside from './DetectClickOutside';
+
+
+type OnFunction = (value?: [number, number] | null) => void;
+export interface RangeSelectorProps{
+  /**
+   * Boolean to toggle time display
+   */
+  dateOnly?: boolean,
+  /**
+   * Display component as dropdown or inline
+   */
+  inline?: boolean,
+  /**
+  * This is a string array that represents the start and end labels of the component
+  */
+  labels?: [string, string]
+  /**
+   * This is a timestamp that represents the maximum date allowed by the component
+   */
+  maxDateTime?: number,
+  /**
+   * This is a timestamp that represents the minimum date allowed by the component
+   */
+  minDateTime?: number,
+  /**
+   * Function to handle onBlur event
+   */
+  onBlur?: OnFunction,
+  /**
+  * Function to handle onChange event
+  */
+  onChange: OnFunction,
+  /**
+   * This is a unix timestamp, which is the number of seconds that have elapsed since Unix epoch
+   */
+  timeStamps?: [number, number],
+  /**
+   * This is must be a IANA time zone string
+   */
+  timeZone?: string,
+}
+
+const RangeSelector: React.FC<RangeSelectorProps> = (({
+  dateOnly = false,
+  maxDateTime = getFutureDate({ years: 1 }),
+  minDateTime = 0,
+  labels = ['From', 'To'],
+  onChange,
+  timeStamps,
+  timeZone = 'Europe/Zurich',
```

Comment:
Use the [default](https://stackoverflow.com/questions/6939685/get-client-time-zone-from-browser) local time zone of user's web client.

Reply:
The decision now is that we use a time zone prop, if there is one else we get it from the browser

Reply:
✅